File: compact-net/CompactNetTupleEngine/FloatToken.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace it.mintlab.mobilenet.tupleengine
{
    class FloatToken : IToken
    {
        private float valueFloat;


        public FloatToken(float number)
        {
            this.valueFloat = number;
        }

        public float getValue()
        {
            return valueFloat;
        }

        public override string ToString()
        {
            return valueFloat.ToString();
        }
    }
}

File: trunk/java/JavaTupleEngine/src/it/mintlab/desktopjava/tupleengine/FloatToken.java
package it.mintlab.desktopjava.tupleengine;

	public class FloatToken implements IToken
	{
		private float valueFloat;


		public FloatToken(float number)
		{
			this.valueFloat = number;
		}

		public final float getValue()
		{
			return valueFloat;
		}

		@Override
		public String toString()
		{
			return (new Float(valueFloat)).toString();
		}
	}